outlines/fsm/json_schema.py

```diff
@@ -105,8 +110,16 @@ def to_regex(resolver: Resolver, instance: dict):
         An object that resolves references to other instances within a schema
     instance
         The instance to translate
+    multiple_ws
+        determines whether to include multiple consecutive whitespace in JSON syntax
     """
 
+    # determine whitespace pattern
+    if multiple_ws:
+        whitespace_pattern = WHITESPACE
```

Comment:
Pass `whitespace_pattern` as an argument directly. This give fine-grained control without having to patch the `WHITESPACE` variable.

Reply:
If we do this, I think it makes sense to validate that their pattern is a proper subset of the legal json whitespace characters `[\t\r\n ]*`. Fortunately that's exactly what `interegular` is designed to do.

Reply:
I think it's completely fine to let users do what they want here; it is not worth the extra code.